Save-System/Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    public int level = 4;
+    public int health = 40;
```

Comment:
Maak hier privates van en zet er [SerializeField] boven zodat ze ook in de editor aan te passen zijn. 
(SerializeField heeft de preference volgens de [styleguide](https://docs.google.com/document/d/1k78hb24HKhIKYgm9eFfgAnLR2skFWFxJwGw6JPDYhC0/edit?usp=sharing))